### PR TITLE
Add option to clear scene

### PR DESCRIPTION
**Pull request by mamueluth**
- [ ] temporary coordinate frames not cleared

### CODE REVIEW

File: sr_moveit2_utils/sr_moveit2_utils/scene_manager.py [hunk 1]
```diff
@@ -486,7 +495,14 @@ def add_objects(self, objects: ObjectDescriptor, as_markers=False) -> list[int]:
                 else:
                     object_mesh_paths.append("")
             else:
+                # if self.object_in_the_scene_storage.get(object_to_add.id, None) is not None:
+                #     self.get_logger().warn(
+                #         f"object_in_the_scene_storage add ob with id {object_to_add.id}"
+                #     )
```
Comment (mamueluth) [inline sr_moveit2_utils/sr_moveit2_utils/scene_manager.py]:
```suggestion
```

File: sr_moveit2_utils/sr_moveit2_utils/scene_manager.py [hunk 2]
```diff
@@ -486,7 +495,14 @@ def add_objects(self, objects: ObjectDescriptor, as_markers=False) -> list[int]:
...
                 self.object_in_the_scene_storage[object_to_add.id] = deepcopy(object_to_add)
+            # self.get_logger().error(
+            #         f"object_in_the_scene_storage{self.object_in_the_scene_storage}"
+            #     )
```
Comment (mamueluth) [inline sr_moveit2_utils/sr_moveit2_utils/scene_manager.py]:
```suggestion
```

File: sr_moveit2_utils/sr_moveit2_utils/scene_manager.py [hunk 3]
```diff
@@ -591,5 +643,6 @@ def publish_as_marker(
             marker.scale.z = 0.04
             marker.type = Marker.TEXT_VIEW_FACING
             marker_array.markers.append(deepcopy(marker))
+
```
Comment (mamueluth) [inline sr_moveit2_utils/sr_moveit2_utils/scene_manager.py]:
```suggestion
```

**Comment by destogl** [comment]
> * [ ]  temporary coordinate frames not cleared

They can not be cleared here in the scene manger.

**Review by destogl — DISMISSED** [review]

File: sr_moveit2_utils/sr_moveit2_utils/scene_manager.py [hunk 4]
```diff
@@ -331,8 +340,8 @@ def detach_object(self, object_id: str, detach_to_link=None):
 
         if not ret:
             self.get_logger().error(f"Detaching object {object_id} has failed!")
-            # re-add the object to the store
-            self.attached_object_store[object_id] = attached_collision_object_to_detach
+            # re-add the object to the scene storage
+            self.object_in_the_scene_storage[object_id] = attached_collision_object_to_detach # pk # changed to scene store
```
Comment (mamueluth) [inline sr_moveit2_utils/sr_moveit2_utils/scene_manager.py]:
```suggestion
            self.attached_object_store[object_id] = attached_collision_object_to_detach store
```

File: sr_moveit2_utils/sr_moveit2_utils/scene_manager.py [hunk 5]
```diff
@@ -331,8 +340,8 @@ def detach_object(self, object_id: str, detach_to_link=None):
 
         if not ret:
             self.get_logger().error(f"Detaching object {object_id} has failed!")
-            # re-add the object to the store
-            self.attached_object_store[object_id] = attached_collision_object_to_detach
+            # re-add the object to the scene storage
+            self.attached_object_store[object_id] = attached_collision_object_to_detach store
```
Comment (mamueluth) [inline sr_moveit2_utils/sr_moveit2_utils/scene_manager.py]:
```suggestion
            self.attached_object_store[object_id] = attached_collision_object_to_detach
```

**Review by destogl — APPROVED** [review]

**Pull request merged**